src/enqueue_links/enqueue_links.js

```diff
@@ -137,7 +139,7 @@ export async function enqueueLinks(options = {}) {
     if (transformRequestFunction) {
         requestOptions = requestOptions.map(transformRequestFunction).filter(r => !!r);
     }
-    const requests = createRequests(requestOptions, pseudoUrlInstances);
+    const requests = createRequests(requestOptions, pseudoUrlInstances).slice(0, limit);
```

Comment:
Would be better to only do the slice if `limit` is defined.

Reply:
Didn't want to create new variables since slice returns original when limit is not defined but will do. Assuming you don't want to change the signature of `createRequests`... what about:

``` javascript
    const requests = limit
        ? createRequests(requestOptions, pseudoUrlInstances).slice(0, limit)
        : createRequests(requestOptions, pseudoUrlInstances);
```

Reply:
Why a new variable?

```js
let requests = createRequests(requestOptions, pseudoUrlInstances);
if (limit) requests = requests.slice(0, limit);
```

Reply:
Okay, sure. `requests` is local so mutability isn't an issue 👍